common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -572,16 +572,13 @@ def test_delete_and_revoke_entitlement(self):
         course_entitlement.refresh_from_db()
         assert course_entitlement.expired_at is not None
 
-    def test_revoke_unenroll_entitlement(self):
-        course_entitlement = CourseEntitlementFactory.create()
+    @patch("entitlements.models.get_course_uuid_for_course")
+    def test_revoke_unenroll_entitlement(self, mock_course_uuid):
```

Comment:
Kindly write doc strings for tests.

Reply:
@Ayub-Khan Almost all tests in this file doesn't have doc strings, how many should I write? Also I'm just fixing these test cases.